packages/inference/src/providers/providerHelper.ts

```diff
@@ -142,6 +144,7 @@ export interface TextToVideoTaskHelper {
 export interface ImageToImageTaskHelper {
 	getResponse(response: unknown, url?: string, headers?: HeadersInit): Promise<Blob>;
 	preparePayload(params: BodyParams<ImageToImageInput & BaseArgs>): Record<string, unknown>;
+	prepareAsyncPayload(args: ImageToImageArgs): Promise<RequestArgs>;
```

Comment:
I would prefer this name
```suggestion
	preparePayloadAsync(args: ImageToImageArgs): Promise<RequestArgs>;
```